webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -167,7 +167,9 @@ const ResourcePoolCard: React.FC<Props> = ({ resourcePool: pool }: Props) => {
               <div className={css.name}>{pool.name}</div>
             </div>
             <div className={css.default}>
-              {(pool.defaultAuxPool || pool.defaultComputePool) && <span>Default</span>}
+              {(pool.defaultAuxPool && pool.defaultComputePool && <span>Default</span>) ||
```

Comment:
What does "pool.defaultAuxPool && pool.defaultComputePool && <span>Default</span>" mean?

Reply:
That means that if the resource pool is both the default aux pool and the default compute pool it evaluates to `<span>Default</span>`.

Reply:
When (pool.defaultAuxPool && pool.defaultComputePool) display Default

Reply:
But isn't that the problem that we're trying to fix "The default compute pool and default aux pool were both labeled "Default". Makes it ambiguous."?

Reply:
The next two lines deal with two "Default" pools:      
          (pool.defaultComputePool && <span>Default Compute</span>) ||
                (pool.defaultAuxPool && <span>Default Aux</span>)}

If two pools are not configured to be the same pool, we label them separately.

Or we could label it "Default Compute & Aux". That makes the label too long. I will leave that decision to the UX team.

Reply:
It would help if in the PR description you mention what the expected behavior should be.

Reply:
Also add a comment to the code describing why it's being done this way.

Reply:
Updated the PR description.

Reply:
`defaultAuxPool` and `defaultComputePool` are both boolean true/false properties of a resource pool. If pool "determined" is the default compute pool but not aux, the object representing it will have defaultAuxPool === false and defaultComputePool === true.

Reply:
> 
> Then how does this test if they are configured to be the same pool?
> 
See resourcepool.pb.go:

```
	// Is this resource pool the default compute pool?
	DefaultComputePool bool `protobuf:"varint,9,opt,name=default_compute_pool,json=defaultComputePool,proto3" json:"default_compute_pool,omitempty"`
	// Is this resource pool the default auxiliary pool?
	DefaultAuxPool bool `protobuf:"varint,10,opt,name=default_aux_pool,json=defaultAuxPool,proto3" json:"default_aux_pool,omitempty"`
```

Reply:
Ok, I see. Thank you.